# redeclare_var_with_var_same_type.py
# This should fail.

x: int = 2
x: int = 3